### DISCUSSION TITLE
Create a chatbot with multi turn human input

### DISCUSSION THREAD

**Discussion by prasenmukh** [microsoft/autogen#4729] [General]
I want to create a support bot where user types in a support query and the support bot responds and  the user can ask follow up questions. Majority of the examples seem to be how to solve a complex user problem in one shot. 
Is there any example where 80% of the time human is involved  after the response.

**Comment by ekzhu** [comment]
Have you looked at the documentation for AgentChat? https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/tutorial/teams.html#pausing-for-user-input

You can put the `run_stream` in a while loop to get human input each time the team stops.

https://microsoft.github.io/autogen/dev/user-guide/agentchat-user-guide/tutorial/swarm.html#customer-support-example

**Comment by tianshangwuyun** [comment]
The really central is a fastapi from delivery account，which after more flexible to talk，is have been relations
